### PR TITLE
Load any model from hub

### PR DESCRIPTION
**Pull request by shon-otmazgin**
We are adding support to load any model and not only `'biu-nlp/f-coref'`.

This behavior was requested by users who trained their own model and want to load it. see [issue](https://github.com/shon-otmazgin/fastcoref/issues/2)

Now, you can init the model like that:
```python
from fastcoref import FCoref

model = FCoref(
   model_name_or_path='your-fast-coref-model-path'
)
```

The coreference head configuration, i.e the size of the ffnn. the pruning ratio, etc. are part now of the model config:
see: https://huggingface.co/biu-nlp/f-coref/blob/main/config.json

I also kept the default to `'biu-nlp/f-coref'`. in this case the model we originally trained is loaded

### CODE REVIEW

File: utilities/coref_dataset.py [hunk 1]
```diff
@@ -8,8 +8,7 @@
 from datasets import Dataset, DatasetDict
 from tqdm import tqdm
 
-from utilities import util
-import consts
+from utilities import util, consts
 from utilities.collate import SegmentCollator, LongformerCollator
```
Comment (ariecattan) [inline utilities/coref_dataset.py:12]: You changed the name of the classes to `LeftOversCollator` and `PadCollator`, no?

File: README.md [hunk 2]
```diff
@@ -123,4 +123,13 @@ python run.py \
...
+After finish training your own model, push the model the huggingface hub (or keep it local), and load your model:
+```python
+from fastcoref import FCoref, CorefArgs
+
+model = FCoref(
+   model_name_or_path='your-fast-coref-model-path'
+)
+```
+
 ## Citation
```
[the remark this answers is not in the document]
Comment (ariecattan) [reply README.md:135]: Also, let's write that we can push and load both `FCoref` and `LingMessCoref`

**Review by ariecattan — COMMENTED** [review]
a few local comments, otherwise looks good :)

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (shon-otmazgin) [reply README.md:135]: Done

File: hard_training/run.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 # sys.path.append(str(Path(__file__).parent.parent))
 
 import sys
-sys.path.append('/home/nlp/shon711/fast-coref')
+sys.path.append('/home/nlp/shon711/fastcoref')
```
[the remark this answers is not in the document]
Comment (shon-otmazgin) [reply hard_training/run.py:8]: done

**Pull request merged**